packages/component-library/src/components/overlay/mt-modal/mt-modal.interactive.stories.ts

```diff
@@ -24,6 +25,26 @@ export const VisualTestContent = {
   }),
 };
 
+export const TestModalOpening = {
```

Comment:
Can you replace that with a unit test? We decided to opt for unit test for everything that is logic related. They run faster and are cheaper than running playwright. For visual related things we have a handful of visual tests and for cases where we rely on things like the `ResizeObserver` we create Storybook tests.

Reply:
Makes sense, I will do it 👍